[2024-12-05T15:54:43.796+0100] {processor.py:186} INFO - Started process (PID=65805) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:54:43.797+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:54:43.797+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:54:43.797+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:54:43.836+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:54:44.578+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:54:44.577+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:54:44.586+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:54:44.630+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:54:44.630+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:54:44.637+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:54:44.637+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:54:44.644+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.849 seconds
[2024-12-05T15:55:16.253+0100] {processor.py:186} INFO - Started process (PID=66702) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:16.255+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:55:16.255+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:16.255+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:16.267+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:55:17.576+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:17.576+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:55:17.583+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:17.622+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:17.622+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:55:17.630+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:17.629+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:55:17.637+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 1.386 seconds
[2024-12-05T15:55:48.867+0100] {processor.py:186} INFO - Started process (PID=67526) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:48.868+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:55:48.869+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:48.869+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:48.877+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:55:49.285+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:49.285+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:55:49.290+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:55:49.329+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:49.329+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:55:49.336+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:55:49.336+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:55:49.342+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.478 seconds
[2024-12-05T15:56:21.505+0100] {processor.py:186} INFO - Started process (PID=68350) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:21.507+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:56:21.507+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:21.507+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:21.523+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:56:22.203+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:22.203+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:56:22.211+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:22.250+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:22.249+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:56:22.256+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:22.256+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:56:22.264+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.763 seconds
[2024-12-05T15:56:53.702+0100] {processor.py:186} INFO - Started process (PID=69158) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:53.710+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:56:53.710+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:53.710+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:53.744+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:56:54.384+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:54.384+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:56:54.392+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:56:54.428+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:54.428+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:56:54.435+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:56:54.435+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:56:54.441+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.740 seconds
[2024-12-05T15:57:25.792+0100] {processor.py:186} INFO - Started process (PID=69927) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:25.797+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:57:25.797+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:25.797+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:25.828+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:57:26.531+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:26.531+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:57:26.539+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:26.577+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:26.577+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:57:26.584+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:26.584+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:57:26.591+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.801 seconds
[2024-12-05T15:57:57.980+0100] {processor.py:186} INFO - Started process (PID=70741) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:57.988+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:57:57.989+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:57.989+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:58.028+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:57:58.692+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:58.692+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:57:58.732+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:57:58.783+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:58.783+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:57:58.794+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:57:58.794+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:57:58.801+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.823 seconds
[2024-12-05T15:58:30.465+0100] {processor.py:186} INFO - Started process (PID=71518) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:58:30.475+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:58:30.475+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:58:30.475+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:58:30.510+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:58:31.149+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:58:31.149+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:58:31.156+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:58:31.195+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:58:31.195+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:58:31.202+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:58:31.202+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:58:31.209+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.745 seconds
[2024-12-05T15:59:02.637+0100] {processor.py:186} INFO - Started process (PID=72284) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:02.646+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:59:02.646+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:02.646+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:02.678+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:59:03.322+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:03.322+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:59:03.354+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:03.398+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:03.398+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:59:03.405+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:03.405+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:59:03.412+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.777 seconds
[2024-12-05T15:59:34.843+0100] {processor.py:186} INFO - Started process (PID=73050) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:34.850+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T15:59:34.851+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:34.851+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:34.881+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T15:59:35.516+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:35.516+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T15:59:35.523+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T15:59:35.580+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:35.580+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T15:59:35.598+0100] {logging_mixin.py:190} INFO - [2024-12-05T15:59:35.597+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T15:59:35.609+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.767 seconds
[2024-12-05T16:00:06.973+0100] {processor.py:186} INFO - Started process (PID=73823) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:06.975+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:00:06.975+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:06.975+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:06.986+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:00:07.616+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:07.616+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:00:07.624+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:07.662+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:07.662+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:00:07.669+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:07.669+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:00:07.676+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.705 seconds
[2024-12-05T16:00:38.988+0100] {processor.py:186} INFO - Started process (PID=74590) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:38.995+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:00:38.995+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:38.995+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:39.029+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:00:39.759+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:39.759+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:00:39.766+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:00:39.802+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:39.802+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:00:39.809+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:00:39.809+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:00:39.816+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.829 seconds
[2024-12-05T16:01:11.188+0100] {processor.py:186} INFO - Started process (PID=75406) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:11.189+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:01:11.190+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:11.190+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:11.201+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:01:11.771+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:11.771+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:01:11.778+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:11.815+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:11.815+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:01:11.823+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:11.823+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:01:11.830+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.644 seconds
[2024-12-05T16:01:43.148+0100] {processor.py:186} INFO - Started process (PID=76175) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:43.149+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:01:43.150+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:43.150+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:43.161+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:01:43.809+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:43.809+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:01:43.816+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:01:43.866+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:43.866+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:01:43.873+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:01:43.873+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:01:43.881+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.735 seconds
[2024-12-05T16:02:15.291+0100] {processor.py:186} INFO - Started process (PID=76938) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:15.292+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:02:15.293+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:15.293+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:15.304+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:02:15.973+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:15.972+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:02:15.980+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:16.017+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:16.017+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:02:16.024+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:16.024+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:02:16.031+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.742 seconds
[2024-12-05T16:02:47.486+0100] {processor.py:186} INFO - Started process (PID=77708) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:47.498+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:02:47.498+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:47.498+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:47.512+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:02:48.116+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:48.116+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:02:48.123+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:02:48.160+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:48.160+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:02:48.167+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:02:48.167+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:02:48.174+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.690 seconds
[2024-12-05T16:03:19.593+0100] {processor.py:186} INFO - Started process (PID=78481) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:19.595+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:03:19.596+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:19.596+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:19.607+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:03:20.212+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:20.212+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:03:20.219+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:20.256+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:20.256+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:03:20.263+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:20.263+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:03:20.269+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.682 seconds
[2024-12-05T16:03:51.585+0100] {processor.py:186} INFO - Started process (PID=79250) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:51.586+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:03:51.587+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:51.587+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:51.599+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:03:52.210+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:52.210+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:03:52.217+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:03:52.254+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:52.254+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:03:52.261+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:03:52.261+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:03:52.268+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.685 seconds
[2024-12-05T16:04:23.894+0100] {processor.py:186} INFO - Started process (PID=80027) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:23.902+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:04:23.902+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:23.902+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:23.940+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:04:24.633+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:24.633+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:04:24.641+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:24.676+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:24.676+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:04:24.683+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:24.683+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:04:24.691+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.798 seconds
[2024-12-05T16:04:55.992+0100] {processor.py:186} INFO - Started process (PID=80832) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:55.994+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:04:55.994+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:55.994+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:56.024+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:04:56.639+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:56.639+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:04:56.646+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:04:56.700+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:56.700+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:04:56.715+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:04:56.715+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:04:56.727+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.739 seconds
[2024-12-05T16:05:28.034+0100] {processor.py:186} INFO - Started process (PID=81585) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:05:28.035+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:05:28.036+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:05:28.036+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:05:28.050+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:05:28.659+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:05:28.659+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:05:28.666+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:05:28.702+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:05:28.702+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:05:28.709+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:05:28.709+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:05:28.717+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.684 seconds
[2024-12-05T16:06:00.198+0100] {processor.py:186} INFO - Started process (PID=82342) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:00.200+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:06:00.200+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:00.200+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:00.212+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:06:01.009+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:01.008+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:06:01.016+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:01.052+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:01.052+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:06:01.059+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:01.059+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:06:01.066+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.869 seconds
[2024-12-05T16:06:32.828+0100] {processor.py:186} INFO - Started process (PID=83128) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:32.829+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:06:32.830+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:32.830+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:32.843+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:06:33.545+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:33.545+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:06:33.553+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:06:33.590+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:33.590+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:06:33.597+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:06:33.597+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:06:33.605+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.779 seconds
[2024-12-05T16:07:04.877+0100] {processor.py:186} INFO - Started process (PID=83932) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:04.880+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:07:04.880+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:04.880+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:04.890+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:07:05.478+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:05.477+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:07:05.484+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:05.521+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:05.521+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:07:05.528+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:05.528+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:07:05.535+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.660 seconds
[2024-12-05T16:07:36.827+0100] {processor.py:186} INFO - Started process (PID=84717) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:36.830+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:07:36.830+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:36.830+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:36.844+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:07:37.424+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:37.424+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:07:37.431+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:07:37.470+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:37.470+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:07:37.478+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:07:37.478+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:07:37.486+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.660 seconds
[2024-12-05T16:08:08.943+0100] {processor.py:186} INFO - Started process (PID=85505) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:08.946+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:08:08.946+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:08.946+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:08.956+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:08:09.718+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:09.718+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:08:09.726+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:09.772+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:09.772+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:08:09.781+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:09.781+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:08:09.792+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.850 seconds
[2024-12-05T16:08:41.185+0100] {processor.py:186} INFO - Started process (PID=86315) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:41.186+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:08:41.187+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:41.187+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:41.197+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:08:41.862+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:41.862+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:08:41.869+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:08:41.920+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:41.920+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:08:41.928+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:08:41.927+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:08:41.935+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.752 seconds
[2024-12-05T16:09:13.879+0100] {processor.py:186} INFO - Started process (PID=87074) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:13.881+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:09:13.882+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:13.881+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:13.893+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:09:14.531+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:14.531+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:09:14.539+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:14.575+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:14.575+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:09:14.590+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:14.590+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:09:14.597+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.719 seconds
[2024-12-05T16:09:46.022+0100] {processor.py:186} INFO - Started process (PID=87871) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:46.024+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:09:46.024+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:46.024+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:46.037+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:09:46.667+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:46.667+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:09:46.674+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:09:46.716+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:46.715+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:09:46.723+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:09:46.723+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:09:46.731+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.711 seconds
[2024-12-05T16:10:18.126+0100] {processor.py:186} INFO - Started process (PID=88656) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:18.127+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:10:18.128+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:18.128+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:18.144+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:10:19.171+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:19.170+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:10:19.179+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:19.218+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:19.218+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:10:19.226+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:19.226+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:10:19.234+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 1.109 seconds
[2024-12-05T16:10:50.547+0100] {processor.py:186} INFO - Started process (PID=89476) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:50.549+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:10:50.549+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:50.549+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:50.561+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:10:51.154+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:51.154+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:10:51.177+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:10:51.217+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:51.217+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:10:51.224+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:10:51.224+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:10:51.231+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.685 seconds
[2024-12-05T16:11:22.556+0100] {processor.py:186} INFO - Started process (PID=90249) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:22.558+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:11:22.559+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:22.559+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:22.576+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:11:23.259+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:23.259+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:11:23.266+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:23.304+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:23.304+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:11:23.311+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:23.311+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:11:23.319+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.765 seconds
[2024-12-05T16:11:54.828+0100] {processor.py:186} INFO - Started process (PID=90998) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:54.838+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:11:54.838+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:54.838+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:54.875+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:11:55.549+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:55.549+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:11:55.558+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:11:55.598+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:55.598+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:11:55.606+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:11:55.606+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:11:55.614+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.787 seconds
[2024-12-05T16:12:27.680+0100] {processor.py:186} INFO - Started process (PID=91761) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:12:27.682+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:12:27.682+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:12:27.682+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:12:27.696+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:12:28.297+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:12:28.297+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:12:28.307+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:12:28.348+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:12:28.348+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:12:28.356+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:12:28.356+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:12:28.363+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.685 seconds
[2024-12-05T16:13:00.678+0100] {processor.py:186} INFO - Started process (PID=92543) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:00.684+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:13:00.685+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:00.684+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:00.748+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:13:01.719+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:01.719+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:13:01.727+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:01.768+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:01.768+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:13:01.776+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:01.776+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:13:01.784+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 1.108 seconds
[2024-12-05T16:13:33.546+0100] {processor.py:186} INFO - Started process (PID=93442) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:33.547+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:13:33.548+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:33.547+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:33.560+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:13:34.163+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:34.163+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:13:34.171+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:13:34.215+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:34.215+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:13:34.224+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:13:34.224+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:13:34.232+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.688 seconds
[2024-12-05T16:14:05.548+0100] {processor.py:186} INFO - Started process (PID=94215) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:05.550+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:14:05.550+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:05.550+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:05.561+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:14:06.178+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:06.178+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:14:06.185+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:06.220+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:06.220+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:14:06.227+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:06.227+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:14:06.234+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.687 seconds
[2024-12-05T16:14:37.762+0100] {processor.py:186} INFO - Started process (PID=94975) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:37.770+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:14:37.771+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:37.771+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:37.810+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:14:38.648+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:38.647+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:14:38.656+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:14:38.702+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:38.702+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:14:38.721+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:14:38.720+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:14:38.734+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.979 seconds
[2024-12-05T16:15:10.328+0100] {processor.py:186} INFO - Started process (PID=95772) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:10.332+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:15:10.333+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:10.333+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:10.356+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:15:11.065+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:11.064+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:15:11.083+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:11.181+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:11.181+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:15:11.189+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:11.189+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:15:11.197+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.871 seconds
[2024-12-05T16:15:42.942+0100] {processor.py:186} INFO - Started process (PID=96566) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:42.947+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:15:42.948+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:42.948+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:42.999+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:15:44.383+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:44.383+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:15:44.392+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:15:44.439+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:44.439+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:15:44.449+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:15:44.449+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:15:44.466+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 1.525 seconds
[2024-12-05T16:16:17.242+0100] {processor.py:186} INFO - Started process (PID=97358) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:17.559+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:16:17.561+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:17.561+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:17.671+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:16:19.832+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:19.832+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:16:19.840+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:19.882+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:19.882+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:16:19.890+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:19.890+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:16:19.898+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 2.658 seconds
[2024-12-05T16:16:51.394+0100] {processor.py:186} INFO - Started process (PID=98257) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:51.399+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:16:51.399+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:51.399+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:51.418+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:16:52.033+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:52.033+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:16:52.042+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:16:52.104+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:52.104+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:16:52.117+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:16:52.117+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:16:52.124+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 0.731 seconds
[2024-12-05T16:17:23.783+0100] {processor.py:186} INFO - Started process (PID=99071) to work on /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:17:23.789+0100] {processor.py:914} INFO - Processing file /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py for tasks to queue
[2024-12-05T16:17:23.789+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:17:23.789+0100] {dagbag.py:588} INFO - Filling up the DagBag from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:17:23.824+0100] {logging_mixin.py:190} INFO - Current Python Path: ['/Users/sijibomijoshua/airflow/airflow_env/bin', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python311.zip', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11', '/Users/sijibomijoshua/.pyenv/versions/3.11.6/lib/python3.11/lib-dynload', '/Users/sijibomijoshua/airflow/airflow_env/lib/python3.11/site-packages', '/Users/sijibomijoshua/airflow/config', '/Users/sijibomijoshua/airflow/plugins', '/Users/sijibomijoshua/airflow/dags']
[2024-12-05T16:17:24.911+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:17:24.911+0100] {main.py:26} WARNING - Python-dotenv could not parse statement starting at line 4
[2024-12-05T16:17:24.918+0100] {processor.py:925} INFO - DAG(s) 'data_warehouse_etl_standard' retrieved from /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py
[2024-12-05T16:17:24.974+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:17:24.974+0100] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-05T16:17:24.992+0100] {logging_mixin.py:190} INFO - [2024-12-05T16:17:24.991+0100] {dag.py:4180} INFO - Setting next_dagrun for data_warehouse_etl_standard to 2024-12-05 00:00:00+00:00, run_after=2024-12-06 00:00:00+00:00
[2024-12-05T16:17:25.007+0100] {processor.py:208} INFO - Processing /Users/sijibomijoshua/airflow/dags/data_warehouse_etl_dag.py took 1.225 seconds
